partner/partner-management-service/src/main/java/io/mosip/pms/config/PartnerManagerDataSourceConfig.java

```diff
@@ -23,16 +20,11 @@
 		basePackages = "io.mosip.pms.device.authdevice.repository.*, io.mosip.pms.common.repository.*",
```

Comment:
remove` io.mosip.pms.common.repository.*` from basePackages

Reply:
add `@EntityScan(basePackageClasses = { DeviceDetail.class })`

Reply:
revert this file to https://github.com/mosip/partner-management-services/blob/develop-java21/partner/partner-management-service/src/main/java/io/mosip/pms/config/PartnerManagerDataSourceConfig.java